lib/webvalve/fake_service_config.rb

```diff
@@ -7,9 +7,12 @@ def initialize(service_class_name:, url: nil)
       @custom_service_url = url
     end
 
-    def should_intercept?
-      WebValve.env.test? || # always intercept in test
-        (WebValve.enabled? && !service_enabled_in_env?)
+    def explicitly_enabled?
+      value_from_env.present? && WebValve::ENABLED_VALUES.include?(value_from_env.to_s)
+    end
+
+    def explicitly_disabled?
+      value_from_env.present? && WebValve::DISABLED_VALUES.include?(value_from_env.to_s)
```

Comment:
If I pass `SERVICE_ENABLED=TRUE`, the way this is written, both `explicitly_enabled?` and `explicitly_disabled?` will return `false`, right?

This might be why ActiveModel only includes FALSE_VALUES -- everything else (aside from `nil` and "") means true. https://github.com/rails/rails/blob/master/activemodel/lib/active_model/type/boolean.rb#L17

Reply:
i have some explicit tests somewhere in here that show that a value like `"this"` is not considered truthy and i think the same for falsey for these methods. i'm happy to change that behavior, but what i was going for here is that the env variable being set doesn't do anything unless it's properly set to a true-like value or a false-like value. thoughts?

Reply:
Got it -- so we say that `SERVICE_ENABLED=something_invalid` is neither explicitly enabled, nor explicitly disabled -- I buy it, but it doesn't feel like a super necessary distinction (like, as an end-developer I wouldn't be surprised if `SERVICE_ENABLED=banana` got cast to `true`) and my more general inclination would be to bool-cast ENV vars the same way everywhere and reduce the local complexity in basically any gem I write. (Like, normally I'd rely on ActiveModel's Boolean type, but that's not available in every lib. I wish Ruby had a built-in shorthand for boolean ENV vars.)

Reply:
Yea. I'm down to change course in this PR. So what casts to false? Only "", nil, "f", "false", false, "0", and 0?

Reply:
Yup, seems reasonable. Since this is only stringy I don't think you need `false` and `0`, unless you are writing a more generic caster.